pcsx2-qt/Settings/SettingsWindow.h

```diff
@@ -72,7 +72,7 @@ class SettingsWindow final : public QWidget
 	__fi AchievementSettingsWidget* getAchievementSettingsWidget() const { return m_achievement_settings; }
 	__fi AdvancedSettingsWidget* getAdvancedSettingsWidget() const { return m_advanced_settings; }
 
-	void registerWidgetHelp(QObject* object, QString title, QString recommended_value, QString text);
+	void registerWidgetHelp(QObject* object, QString const title, const QString text, const QString recommended_value = "");
```

Comment:
why not `const QString&` (i.e. by reference)? although it probably won't make much of a difference in practice since these strings are usually returned by value from `tr()`...